ractor_actors/src/watchdog/mod.rs

```diff
@@ -0,0 +1,75 @@
+use crate::watchdog::watchdog::WatchdogMsg;
```

Comment:
Can we get some kind of docs on what this does? How it's working, what it's monitoring/etc.

Reply:
Biggest question, how does the watchdog "watch" actors? What is it doing to ping them or check they're alive?